progs/tests/tuple/basic.sy

```diff
@@ -6,9 +6,9 @@ start :: fn {
     b := (1,)
     b[0] <=> 1
 
-    empty := ()
+    empty := (,)
     c := (empty,)
-    c[0] <=> ()
+    c[0] <=> (,)
```

Comment:
I disagree with `(,)` being the syntax for empty tuples. The way I see it, commas can only follow elements in a tuple.